prices_voice.go

```diff
@@ -20,11 +20,20 @@ type NumberVoicePriceService struct {
 	client *Client
 }
 
+func numberVoicePricePart(number, originationNumber string) string {
+	res := number
+	if originationNumber != "" {
+		res += "?OriginationNumber=" + originationNumber
+	}
+	return res
+}
+
 type PrefixPrice struct {
```

Comment:
This change is going to break everyone who currently relies on this code. How should we handle this?

Reply:
as a variant we can make v2 branch for v2 api version

Reply:
Huh. TBH I haven't been too happy with that format. I think maybe we should just upgrade the library version to 2.0.

Can you update CHANGELOG.md with the required changes if you have code that's working with v1, to get it working again with v2?

Reply:
done